batch/README.md

```diff
@@ -19,18 +26,37 @@ minikube start
 
 When you want to return to using a google k8s cluster, you can run this:
 
-```
-gcloud container clusters get-credentials CLUSTER_NAME
-```
+```sh
+# Login with a Google account that is linked to a Google Cloud project that manages the cluster of interest
+gcloud auth login
 
-Set some environment variables so that docker images are placed in the
-`minikube` cluster's docker registry:
+gcloud projects list
+# Shows
+#PROJECT_ID             NAME             PROJECT_NUMBER
+#my-awesome-project     xxxx             xxxx
```

Comment:
I'd put the description of what the shell block does as text outside the block, then in the block, I think we should stick to posix-y looking shell, i.e.:
```
$ gcloud projects list
PROJECT_ID             NAME             PROJECT_NUMBER
my-awesome-project     xxxx             xxxx
```

Reply:
Sure. I was going for executable commands inside the code block, but see your point.